Velaptor/Factories/IReactableFactory.cs

```diff
@@ -35,6 +35,12 @@ internal interface IReactableFactory
     /// <returns>The reactable.</returns>
     IPushReactable<BatchSizeData> CreateBatchSizeReactable();
 
+    /// <summary>
+    /// Creates a window size reactable.
+    /// </summary>
+    /// <returns>The reactable.</returns>
+    IPushReactable<WindowSizeData> CreateWindowSizeReactable();
+
     /// <summary>
     /// Creates a view port reactable.
```

Comment:
Spelling.  Refactor to make viewport one word.
```suggestion
    /// Creates a viewport reactable.
```